app/views/links/_current_reaction.erb

```diff
@@ -0,0 +1,3 @@
+<%= turbo_frame_tag "link_response_#{link.id}" do %>
+  <%= render 'links/reaction', past_link: link.current_past_link %>
```

Comment:
When you get a chance, can you make 100% sure that `nil` for the `past_link` local is OK?

Reply:
Yeah, it's fine. As long as past_link is defined, it can be nil--the first line of the partial checks that. There are several unpopulated test links on my local test account.